### PR TITLE
[IMP] requirements: fine tuning for distributions

### PR DESCRIPTION
**Pull request by d-fence**
Fine tuning the requirements.txt to better match Ubuntu Noble / Jammy as well as Debian Bookworm.

### CODE REVIEW

**Comment by d-fence** [comment]
reqchecknoble

File: addons/account/tools/certificate.py [hunk 1]
```diff
@@ -50,8 +50,8 @@ def public_numbers():
                 for item in certificate.subject.get_attributes_for_oid(oid)
             ]
         ),
-        not_valid_after=certificate.not_valid_after,
-        not_valid_before=certificate.not_valid_before,
+        not_valid_after=certificate.not_valid_after_utc,
+        not_valid_before=certificate.not_valid_before_utc,
```
Comment (william-andre) [inline addons/account/tools/certificate.py:54]: From the docs, this doesn't exist in `cryptography < 42.0.0`, but we are using a lower version for python < 3.12 :thinking:

Comment (d-fence) [reply addons/account/tools/certificate.py:54]: Thanks nice catch.

**Review by Xavier-Do — APPROVED** [review]

**Comment by Xavier-Do** [comment]
@robodoo r+

**Comment by Xavier-Do** [comment]
@robodoo rebase-ff

**Comment by d-fence** [comment]
robodoo r-

**Comment by d-fence** [comment]
robodoo r+ rebase-ff

**Pull request closed without merging**